verification/README.md

```diff
@@ -0,0 +1,248 @@
+# <h1 align="center"> Verification </h1>
+
+---
+
+A contract verification service. Runs as an HTTP server and allows
+making verification requests through REST API. It is stateless
+and answers requests based on provided information only.
+
+## Building from source
+Install rustup from rustup.rs.
+```
+git clone git@github.com:blockscout/blockscout-rs.git
+
+cd blockscout-rs
+
+cargo build --all --release
+```
+You can find the built binary in `target/release` folder.
+
+## Installing through cargo
+Another way to install the binary without cloning the repository is to use cargo straightway:
+```
+cargo install --git https://github.com/blockscout/blockscout-rs --bin verification
+```
+In that case, you can run the binary using just `verification`.
+
+## Configuration
+Currently, the service uses a configuration file the path to which is specified via CLI flag `--config-path=[path]`.
+The configuration file may contain the following options:
+```toml
+[server]
+# IP address and port number the server should listen to
+addr = "0.0.0.0:8043"
+
+[solidity]
+# if false solidity verification is not supported
+enabled = true
+# list of all available compilers and information about them
+compilers_list_url = "https://raw.githubusercontent.com/blockscout/solc-bin/main/list.json"
+
+[sourcify]
+# if false sourcify verification is not supported
+enabled = true
+# Sourcify API endpoint
+api_url = "https://sourcify.dev/server/"
+# number of failing attempts the server makes to Sourcify API 
+verification_attempts = 3
+# the maximum period the service is waiting for the Sourcify response
+request_timeout = 10
+```
+For all keys omitted from the configuration file default values from the example above are used.
+
+# Api
+
+---
```

Comment:
github already draws the line 

![image](https://user-images.githubusercontent.com/14099557/175980972-e2d1fd44-a03d-41e5-b750-
2765566d59ff.png)